pkg/agent/baker.go

```diff
@@ -1048,3 +1060,72 @@ root = "{{GetDataDir}}"{{- end}}
     address = "/run/teleportd/snapshotter.sock"
 {{- end}}
 `
+
+// this pains me, but to make it respect mutability of vmss tags,
+// we cannot use go templates at runtime.
+// CSE needs to be able to generate the full config, with all params,
+// with the tags pulled from wireserver. this is a hack to avoid
+// moving all the go templates to CSE -- we allow two options,
+// duplicate them in CSE base64-encoded, and pick the right one.
+// they're identical except for GPU runtime class.
+const containerdConfigNoGpuTemplateString = `version = 2
```

Comment:
I guess we can't just add extra parameterization on the existing template since whether or not we want to skip the driver installation is based on the instance data rather than the bootstrapping config right?

Reply:
right -- we can move all of the parameterization itself into bash, but that's even more tedious.

i'd love to write a standalone python/go/or whatever helper for this honestly, just low priority

Reply:
Is it possible to copy this from the original file instead of a full copy-paste? There's no GPU specific difference here right?

Reply:
there is -- I remove a large piece. see L1070 above. no default_runtime_class = "nvidia-container-runtime" or configuration for that at all

I could leave the nvidia container runtime but thee's still no great way to default it